server/routerlicious/packages/protocol-base/src/quorum.ts

```diff
@@ -92,7 +92,12 @@ export class QuorumClients extends TypedEventEmitter<IQuorumClientsEvents> imple
      * Adds a new client to the quorum
      */
     public addMember(clientId: string, details: ISequencedClient) {
-        assert(!this.members.has(clientId), 0x1ce /* clientId not found */);
+        if (this.members.has(clientId)) {
+            throw new Error(
+                `Cannot add member. Client id: ${clientId} already present.`,
```

Comment:
Consider making this a LoggingError with cientId being a property on such object (not baked into error text of the message). This will make it much better in terms of ability to aggregate errors (and notices if there is a spike in such errors, instead of having 1-hits across long tail of distribution curve), but also easier to do various join queries to find other events for this clientId.

Reply:
Yes please!  And same below.